Makefile

```diff
@@ -6,8 +6,8 @@ dev:
 	sh ./scripts/build.sh
 
 bootstrap:
-	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc
-	go install google.golang.org/protobuf/cmd/protoc-gen-go
+	go install -mod mod google.golang.org/grpc/cmd/protoc-gen-go-grpc
+	go install -mod mod google.golang.org/protobuf/cmd/protoc-gen-go
```

Comment:
Thinking whether `-mod mod` is the proper flag to use in this case. Think that `-mod readonly` is a better options since it will not affect the `go.mod` and `go.sum`.

Reply:
nice I didn't know about the 'install modes', I guess we'd need to update some other Makefiles as well :wink: